design/templates/solution_design.md

```diff
@@ -0,0 +1,74 @@
+# Solution Design - Template
+[//]: # (Change the title above from "Template" to your design's title) 
+
+## Table of Contents
+[//]: # (You can use this tool to generate a TOC - https://ecotrust-canada.github.io/markdown-toc/) 
+
+## Glossary
+[//]: # (Describe terms that will be used throughout the design)
+[//]: # (You can use this tool to generate a table - https://www.tablesgenerator.com/markdown_tables#)
+
+| **Term** | **Description** |
+|----------|-----------------|
+|          |                 |
+|          |                 |
+
+## Useful links
+[//]: # (Add links that may be useful for the reader)
+
+## Background
+[//]: # (Give relevant background for the designed feature. What is the motivation for this solution?)
+
+## Issue description
+[//]: # (Elaborate on the issue you are writing a solution for)
+
+## Solution
+[//]: # (Elaborate on the solution you are suggesting in this page. Address the functional requirements and the non functional requirements that this solution is addressing. If there are a few options considered for the solution, mention them and explain why the actual solution was chosen over them. Add an execution plan when relevant. It doesn't have to be a full breaking of the feature, but just a recommendation to how the solution should be approached.)
+
+### Design
+[//]: # (Add any diagrams, charts and explanations about the design aspect of the solution. Elaborate also about what the expected user experience for the feature)
+
+### Backwards compatibility
+[//]: # (Address how you are going to handle backwards compatibility, if necessary)
```

Comment:
is it necessary to have a whole separate section on this?

Reply:
I think it is important enough to have a sub-section under "Solution". It can be interesting to see how the solution handles the backwards-compatibility.